api/envoy/extensions/filters/network/postgres_proxy/v3alpha/postgres_proxy.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.network.postgres_proxy.v3alpha;
+
+import "udpa/annotations/migrate.proto";
+import "udpa/annotations/status.proto";
+import "udpa/annotations/versioning.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.filters.network.postgres_proxy.v3alpha";
+option java_outer_classname = "PostgresProxyProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).work_in_progress = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Postgres proxy]
+// Postgres Proxy :ref:`configuration overview
+// <config_network_filters_postgres_proxy>`.
+// [#extension: envoy.filters.network.postgres_proxy]
+
+message PostgresProxy {
+  // The human readable prefix to use when emitting :ref:`statistics
+  // <config_network_filters_postgres_proxy_stats>`.
+  string stat_prefix = 1 [(validate.rules).string = {min_bytes: 1}];
```

Comment:
We're now asking folks to use `min_len` rather than `min_bytes` (since PGV supports it).